app/controllers/users.rb

```diff
@@ -19,3 +19,36 @@
 get '/users/new' do
   erb :"/users/new"
 end
+
+get '/users/:id' do |id|
+  @user = User.find_by(id: id)
```

Comment:
Nice use of the block parameter, but instead of `find_by` you could simply pass "id" to `User.find`